src/pages/defineGroupPassengers.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { ReactElement } from 'react';
+import InsetText from '../components/InsetText';
+import { getSessionAttribute } from '../utils/sessions';
+import TwoThirdsLayout from '../layout/Layout';
+import { passengerTypesList, GROUP_PASSENGER_TYPES } from '../constants';
+import { ErrorInfo, CustomAppProps, NextPageContextWithSession } from '../interfaces';
+import ErrorSummary from '../components/ErrorSummary';
+import FormElementWrapper from '../components/FormElementWrapper';
+import CsrfForm from '../components/CsrfForm';
+import { GroupPassengerTypesWithErrors, GroupPassengerTypes } from './api/defineGroupPassengers';
+
+const title = 'Define Group Passengers - Fares Data Build Tool';
+const description = 'Group Passengers selection page of the Fares Data Build Tool';
+
+const errorId = 'passenger-type-error';
+
+export type PassengerAttributes = {
+    passengerTypeDisplay: string;
+    passengerTypeValue: string;
+    greyedOut: boolean;
+};
+
+const isGroupPassengerWithErrors = (
+    defineGroupPassengersAttribute: GroupPassengerTypes | GroupPassengerTypesWithErrors,
+): defineGroupPassengersAttribute is GroupPassengerTypesWithErrors =>
+    (defineGroupPassengersAttribute as GroupPassengerTypesWithErrors).errors !== undefined;
+
+interface PassengerTypeProps {
+    groupPassengerInfo: GroupPassengerTypes | GroupPassengerTypesWithErrors;
+}
+
+const insetText = 'More passenger types will become available soon';
+
+const DefineGroupPassengers = ({
+    groupPassengerInfo,
+    csrfToken,
+}: PassengerTypeProps & CustomAppProps): ReactElement => {
+    const errors: ErrorInfo[] = isGroupPassengerWithErrors(groupPassengerInfo) ? groupPassengerInfo.errors : [];
+    return (
+        <TwoThirdsLayout title={title} description={description} errors={errors}>
+            <CsrfForm action="/api/defineGroupPassengers" method="post" csrfToken={csrfToken}>
+                <>
+                    <ErrorSummary errors={errors} />
+                    <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
+                        <fieldset className="govuk-fieldset" aria-describedby="passenger-type-page-heading">
+                            <legend className="govuk-fieldset__legend govuk-fieldset__legend--l">
+                                <h1 className="govuk-fieldset__heading" id="passenger-type-page-heading">
+                                    Select a passenger type
+                                </h1>
+                            </legend>
+                            <span className="govuk-hint" id="passenger-type-hint">
+                                Relate the ticket(s) to a passenger type
+                            </span>
+                            <FormElementWrapper
+                                errors={
+                                    errors.length ? [{ errorMessage: errors[0].errorMessage || '', id: errorId }] : []
```

Comment:
If we pass the errorId in in the api, then we should just be able to pass `errors={errors}` here